crates/index-scheduler/src/lib.rs

```diff
@@ -795,27 +795,29 @@ impl IndexScheduler {
 
             let rtxn = self.env.read_txn()?;
 
-            let task_reader = TaskReader {
-                rtxn: &rtxn,
-                index_scheduler: self,
-                tasks: &mut updated.into_iter(),
-                buffer: Vec::with_capacity(50), // on average a task is around ~100 bytes
-                written: 0,
-            };
-
-            // let reader = GzEncoder::new(BufReader::new(task_reader), Compression::default());
-            let reader = GzEncoder::new(BufReader::new(task_reader), Compression::default());
-            let request = ureq::post(url)
-                .timeout(Duration::from_secs(30))
-                .set("Content-Encoding", "gzip")
-                .set("Content-Type", "application/x-ndjson");
-            let request = match &self.webhook_authorization_header {
-                Some(header) => request.set("Authorization", header),
-                None => request,
-            };
-
-            if let Err(e) = request.send(reader) {
-                tracing::error!("While sending data to the webhook: {e}");
+            for url in &self.webhook_urls {
+                let mut task_reader = TaskReader {
+                    rtxn: &rtxn,
+                    index_scheduler: self,
+                    tasks: updated.iter(),
+                    buffer: Vec::with_capacity(50), // on average a task is around ~100 bytes
```

Comment:
That was some old outdated code 😱 , tasks are way larger today 
```suggestion
                    buffer: Vec::with_capacity(1024),
```

Or maybe even better if we use a [`page_size`](https://docs.rs/page_size/latest/page_size/) because once we want to send it we'll have to allocate a full page size anyway.
```suggestion
                    buffer: Vec::with_capacity(page_size()),
```